### PR TITLE
feature:Use_Spring_IOC_to_refactor_RpcProxyManager

### PR DESCRIPTION
**Pull request by cmgyqjj**
For: Use Spring IOC to refactor RpcProxyManager [#135 ](https://github.com/crossoverJie/cim/issues/135)

In previous versions, RpcProxyManager would be repeatedly created. In this change, the `RpcProxyManager` is managed using the Spring Ioc approach to reuse objects.

### CODE REVIEW

File: cim-forward-route/src/main/java/com/crossoverjie/cim/route/service/impl/AccountServiceRedisImpl.java [hunk 1]
```diff
@@ -55,6 +55,9 @@ public class AccountServiceRedisImpl implements AccountService {
     @Autowired
     private OkHttpClient okHttpClient;
 
+    @Autowired
```
Comment (crossoverJie) [inline cim-forward-route/src/main/java/com/crossoverjie/cim/route/service/impl/AccountServiceRedisImpl.java:58]:
```suggestion
    @Resource
```

We will use `@Resource` uniformly, and the existing `@Autowired` also needs to be replaced.

File: cim-server/src/main/java/com/crossoverjie/cim/server/config/BeanConfig.java [hunk 2]
```diff
@@ -24,6 +26,9 @@
 @Configuration
 public class BeanConfig {
 
+    @Autowired
```
Comment (crossoverJie) [inline cim-server/src/main/java/com/crossoverjie/cim/server/config/BeanConfig.java:29]:
```suggestion
    @Resource
```

File: cim-server/src/main/java/com/crossoverjie/cim/server/kit/RouteHandler.java [hunk 3]
```diff
@@ -32,6 +32,9 @@ public class RouteHandler {
     @Autowired
     private AppConfiguration configuration;
 
+    @Autowired
```
Comment (crossoverJie) [inline cim-server/src/main/java/com/crossoverjie/cim/server/kit/RouteHandler.java:35]:
```suggestion
    @Resource
```

Comment (cmgyqjj) [reply cim-server/src/main/java/com/crossoverjie/cim/server/kit/RouteHandler.java]: Sure, They have been solved.

**Review by crossoverJie — APPROVED** [review]
LGTM

**Pull request merged**